src/entity_recognizer/EntityDictionary.cpp

```diff
@@ -200,7 +200,7 @@ void EntityDictionary::ReadGazetteerFiles() {
           }
           int l = -1;
           for (l = 0; l < gazetteer_word_entity_tags_[word_id].size();
-               ++l) {
+          ++l) {
```

Comment:
put previous indentation back. 

(this is weird by the way.... Is it MSVC that does it automatically?)